packages/clients/src/aptos/gas.ts

```diff
@@ -40,6 +48,7 @@ const Fund = object({
   name: string(),
   balance: integer(),
   inFlight: integer(),
+  depositAddress: optional(string()),
```

Comment:
This only accepts `undefined`, but the API seems to be returning `null`. Can you double check?

Reply:
Ah, good catch, you're right the api does return `null` here if there isn't an address. I have updated the type to `nullable(string())`, let me know if you're looking for anything different here.